public/buttons.html

```diff
@@ -10,19 +10,35 @@
   </head>
   <body>
     <h1>PayPal Buttons Example</h1>
-    <div id="paypal-buttons-container"></div>
+    <div id="paypal-button-container" style="max-width: 440px"></div>
     <script>
       function loadPayPalSDK() {
         return fetch("/api/paypal/client-config")
           .then((response) => response.json())
-          .then(({ clientID, webBaseUrl }) => {
+          .then(({ clientID, webBaseUrl, currency, intent }) => {
             return paypalLoadScript({
+              // Supported query parameters: https://developer.paypal.com/sdk/js/configuration/#link-queryparameters
               "client-id": clientID,
               sdkBaseURL: `${webBaseUrl}/sdk/js`,
+              intent: intent?.toLowerCase(),
+              currency: currency,
+              // "enable-funding": "venmo", // Venmo button will only render to payers with a US IP and currency=USD transactions
+              // "buyer-country": "US", // Simulate a differnet geographic IP to see which buttons will display there. May only be used in sandbox mode.
+              // "locale": "en_US", // Render buttons in a different language--by default the browser language is detected and used.
             });
           });
       }
 
+      // Example function to show a result to the user. Your site's UI library can be used instead,
+      // however alert() should not be used as it will interrupt the JS SDK window
+      function resultMessage(message, hideButtons = false) {
```

Comment:
Could we pass an option in here instead? That way we don't have to rely on parameter order and can add more params in the future if need be.
```suggestion
      function resultMessage({ message, hideButtons = false }) { }
```

Reply:
We could yes, I did consider it! -- but it's sort of a throwaway sample function in lieu of nothing better and I opted to keep the usual invocation `resultMessage("....")` with a single string as short as I could to make the callback functions as easy to read as humanly possible (for novices completely new to this)

Reply:
added options but kept message as first param